src/styles/reset.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+}
```

Comment:
Using the `*` selector for resetting styles is generally not recommended, as it applies the styles globally to all elements and can lead to unintended consequences and conflicts with other styles in your code.
Leave only box-sizing here to ensure a consistent box model throughout your page.